dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -52,14 +55,23 @@ public void basicRecordTest() throws Exception {
     public void recordValidatePropertyAndValueTest() throws Exception {
         // TODO create a test using the Validator.validateProperty and Validator.validateValue methods
         // on a record
+        Person propertydata = new Person(null);
+        Set<ConstraintViolation<Person>> propertyViolations = validator.validateProperty(propertydata, "name");
+        Set<ConstraintViolation<Person>> valueViolations = validator.validateValue(Person.class, "name", null);
+        assertTrue("Record Person(null) should have validated with one violation", propertyViolations.size() == 1);
+        assertEquals(1, valueViolations.size());
```

Comment:
This should have a message like the assertTrue above it to explain in the case of failure

Reply:
updated